libs/sdk-backend-spi/src/workspace/dashboards/analyticalWidgets.ts

```diff
@@ -84,6 +84,18 @@ export interface IInsightWidgetBase extends IAnalyticalWidget {
      * Drill interactions configured for the insight widget.
      */
     readonly drills: InsightDrillDefinition[];
+
+    /**
+     * Configuration of the widget itself regardless of the visualization type
+     */
+    readonly configuration: IInsightConfiguration;
+}
+
+/**
+ * @alpha
+ */
+export interface IInsightConfiguration {
```

Comment:
Should this be called `IInsightWidgetConfiguration` instead since it is configuring the widget, not the insight?